buildspec.yml

```diff
@@ -0,0 +1,33 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - sudo apt-get update
+      - sudo apt-get install apt-transport-https
+      - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - sudo apt-get update && sudo apt-get install yarn
+  pre_build:
+    commands:
+      - yarn install
+  build:
+    commands:
+      - yarn build:all
+      - ./node_modules/mocha/bin/mocha -r ts-node/register test/**/*.spec.ts
+  post_build:
+    commands:
+      - >-
+        if [ $CODEBUILD_SOURCE_VERSION == master ]; then
+        yarn add global serverless && serverless deploy; fi
+      - zip -r build.zip *
```

Comment:
Can you clarify this?  I don't follow why we're zipping the `build` directory and saving it as an artifact.

Reply:
Ah, well maybe this isn't necessary after all. I noticed that serverless zips the entire directory for us and stores it. I figured maybe it would be a good idea to duplicate that effort, but put it in a more human-readable directory. 

Then again, this may just be redundant. We could also just make the packages serverless bundles more readable by specifying a custom directory in the `serverless.yml` file.

Reply:
Ah, okay, in that case I think we can do without it.  That'll save on some build time and data transfer.